src/amo/components/Errors/AuthExpired/index.js

```diff
@@ -0,0 +1,69 @@
+/* @flow */
+/* global window */
+import * as React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+import translate from 'amo/i18n/translate';
+import Button from 'amo/components/Button';
+import type { I18nType } from 'amo/types/i18n';
+import { logOutUser as logOutUserAction } from 'amo/reducers/users';
+import type { DispatchFunc } from 'amo/types/redux';
+
+type Props = {||};
+
+type DefaultProps = {|
+  _window: typeof window,
+|};
+
+type InternalProps = {|
+  ...Props,
+  ...DefaultProps,
+  i18n: I18nType,
+  dispatch: DispatchFunc,
+|};
+
+export class AuthExpiredBase extends React.Component<InternalProps> {
+  static defaultProps: DefaultProps = {
+    _window: typeof window !== 'undefined' ? window : {},
+  };
+
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(logOutUserAction());
+  }
+
+  render(): React.Node {
+    const { _window, i18n } = this.props;
+
+    const reloadButton = (
+      <Button
+        buttonType="action"
+        micro
+        onClick={() => _window.location.reload()}
```

Comment:
Ah, I looked at the report but couldn't see where it actually pointed to the uncovered line

Reply:
It takes a bit of getting used to, and sometimes it's hard to find, but in this case just looking at the diff (https://app.codecov.io/gh/mozilla/addons-frontend/compare/11092/diff), and scrolling down, you can see the missing coverage highlighted in red.

Reply:
I have a test based on the one for `ErrorList` but the last line, `sinon.assert.called`, fails saying it's not called
```typescript
it('renders a reload button', () => {
    const _window = { location: { reload: sinon.stub() } };
    const root = render();

    const button = root.find(Button);
    expect(button.childAt(0).text()).toContain('Reload the page');
    expect(button.prop('onClick')).toBeDefined();

    const action = button.prop('onClick');
    // Simulate the callback on button press.
    action();

    // The button should reload the location.
    sinon.assert.called(_window.location.reload);
  });
```

Reply:
It's not working because you are not passing the `_window` mock that you create into the `render` method of the component, so when it executes it doesn't use your `_window` mock.